android/lib/pages/profile_page.dart

```diff
@@ -588,7 +601,15 @@ class _ProfilePageState extends State<ProfilePage> {
                                                   user_account_info
                                                               .profile_picture_id ==
                                                           null
-                                                      ? profilePictureBuilder(3)
+                                                      ? CircleAvatar(
```

Comment:
Do not use those. Instead use `circleAvatarBuilder()` in `get_image_builder.dart` as we discussed. There are a lot of problems with image processing in this code.